GirafRest.Test/appsettings.Development.json

```diff
@@ -1,7 +1,7 @@
 ﻿{
     "_Comment": "ONLY FOR TESTING PURPOSE",
     "ConnectionStrings": {
-        "DefaultConnection": "server=localhost;port=3306;userid=user;password=password;database=giraf;Allow User Variables=True"

```

Comment:
Also, it is intentional to change user -> root here?

Reply:
Fixed.